src/main/python/debugFilter.py

```diff
@@ -0,0 +1,179 @@
+import sys
```

Comment:
Would it make sense to document this script, too?

Reply:
I was seeing this as a throw-away script, but, you're right, If I'm adding this to source control, who knows what will happen, so I should document it.